### DISCUSSION TITLE
many different authors (same series)

### DISCUSSION THREAD

**Discussion by mobergmann** [advplyr/audiobookshelf#2153] [Q&A] [answered]
I have some audio-books, all belonging to the same series. But some of the episodes have different authors. How can I add the audio-books to my library, so that the metadata can be fetched? 
I tries some variants of the [default directory structure](https://www.audiobookshelf.org/docs/#book-directory-structure):

---

Layout 1:
```
├── André Marx
│   └── Feuermond
│        └── Audio File.mp3
├── André Marx, André Minninger
│    └── Spur ins Nichts
│        └── Audio File.mp3
├── N.N.
│    ├── Angriff der Computer-Viren
│    │   └── Audio File.mp3
│    ├── Auf tödlichem Kurs
│    │   └── Audio File.mp3
```
The problem with this layout is, that the metadata for the episodes is found (if I am correct), but the episodes doesn't get sorted to their series.

---

Layout 2:
```
├── André Marx
│   └── Die drei ???
│       └── Feuermond
│           └── Audio File.mp3
├── André Marx, André Minninger
│   └── Die drei ???
│       └── Spur ins Nichts
│           └── Audio File.mp3
├── N.N.
│   └── Die drei ???
│       ├── Angriff der Computer-Viren
│       │   └── Audio File.mp3
│       ├── Auf tödlichem Kurs
│       │   └── Audio File.mp3
```
The issue with this layout is, that the metadata is not fetched correctly, meaning that the title of the episode is not displayed correctly.

---

Is there a solution for this problem?

**Comment by mobergmann** [comment]
Okay, maybe I fixed it by setting the metadata provider to `Audible.de`.

**Comment by mobergmann** [comment]
my issue now is, that the series is not sorted

**Comment by nichwall** [comment]
When you match the metadata to audible.de, are the books just being assigned a series or do they also have a series sequence number? They need a number to be sorted correctly, but sometimes online providers don't give you that information.

**Reply by mobergmann** [reply, marked as answer]
I fixed it by using Layout 2. The series is still not sorted, but I matched the files manually and I can now search for the subtitle.
